stories/90-Components/Organisms/Site Header.stories.mdx

```diff
@@ -393,3 +393,101 @@ On mobile, the majority of components in the Site Header will be hidden as to fi
     }
   }}</Story>
 </Preview>
+
+<Preview withToolbar={true}>  
+  <Story name="Header Config 4 (No Menu)" height="150px">{{
+    template: hbs`
+          <NyprOHeader
+                  @showMenuButton=false
+                  @donateURL={{DONATE_URL}}
+                  data-test-main-header
+                  as |header|
+          >
+            <header.left data-test-header-left as |left|>
```

Comment:
With the change suggested above this part of the example would become
`<header.left @showMenuButton=false as |left|>`

Reply:
If we make this change, we'd have to set the showMenuButton=true or =false for all the headers

Reply:
We can have a default, the same as if we set it in the top level.